gems/aws-sdk-core/lib/aws-sdk-core.rb

```diff
@@ -75,6 +75,8 @@ module Aws
   # dynamic endpoints
   autoload :Endpoints, 'aws-sdk-core/endpoints'
 
+  autoload :Sigv4, 'aws-sigv4'
```

Comment:
Why is this not in the sigv4 gem? Or why even autoload this at all?

Reply:
We were previously requiring the aws-sigv4 gem from core, so this keeps that functionality.